test/11_array.jl

```diff
@@ -132,6 +132,38 @@ for (A, V, M) in ((NominalArray, NominalVector, NominalMatrix),
         @test x[3] === x.pool.valindex[1]
         @test levels(x) == ["e", "a", "b", "c"]
 
+        x = V{String, R}(a)
+        push!(x, "a")
+        @test length(x) == 4
+        @test String(x[end]) == "a"
```

Comment:
No need for `String` here, `==` is implemented to allow direct comparisons with `NominalValue`.